readme.md

```diff
@@ -1,115 +1,96 @@
+This repository contains code used to generate synthetic LAR files. Currently, the test files repository supports file creation for submission years, 2018 and 2019. The success of the HMDA program requires that the data submitted meet the technical and business requirements of the HMDA Platform. The code and test files in this repository will provide a standardized means of checking the implementation of business rules for data submission in the HMDA Platform.
 
 ## Repository Use Cases:
-- Enable business analysts to test the 2018 HMDA Platform
+- Enable business analysts to test the HMDA Platform. 
 - Provide files to check API returns
 - Provide a resource for developers to use while implementing business rules in code
-- Provide synthetic data files meeting the 2018 HMDA schema to be used in training
+- Provide synthetic data files meeting HMDA schemas for 2018 and 2019 to be used in training
 
 ## Repository Structure
-- [Python](https://github.com/cfpb/hmda-test-files/tree/master/python):
+Each year listed in the parent directory contains its own codebase for running test files for that submission year. Users should navigate to each year directory to run scripts and view code.  
+- [2018/python](https://github.com/cfpb/hmda-test-files/tree/master/2018/python) and [2019/python](https://github.com/cfpb/hmda-test-files/tree/master/2019/python):
     - Contains the notebooks and python scripts used to generate synthetic LAR data, perturb the data so that it fails specific business rules, and code to check that the produced files fail edits as expected.
 
-- [Schemas](https://github.com/cfpb/hmda-test-files/tree/master/schemas)
-    - Contains JSON objects that represent the data structures for LAR and Transmittal Sheet as defined by the [2018 HMDA FIG](https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-hmda-fig-2018-hmda-rule.pdf)
+- [2018/schemas](https://github.com/cfpb/hmda-test-files/tree/master/2018/schemas) and [2019/schemas](https://github.com/cfpb/hmda-test-files/tree/master/2019/schemas)
+    - Contains JSON objects that represent the data structures for LAR and Transmittal Sheet as defined by the [2018 HMDA FIG](https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-hmda-fig-2018-hmda-rule.pdf) and the [2019 HMDA FIG](https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2019-hmda-fig.pdf)
 
-- [Dependencies](https://github.com/cfpb/hmda-test-files/tree/master/dependencies)
-    - Contains files used in the generation of synthetic LAR data.
-        - Tract to CBSA data for 2015
+- [2018/dependencies](https://github.com/cfpb/hmda-test-files/tree/master/2018/dependencies) and [2019/dependencies](https://github.com/cfpb/hmda-test-files/tree/master/2019/dependencies)
+    	- Contains files used in the generation of synthetic LAR data.
+        - Tract to CBSA data.
         - A file containing a list of US ZIP codes
 
+To read further about the file generation features for each year, please refer to the documents below:
```

Comment:
change this to: `the links below contain a discussion of file generation features by year`